--- lib/ssqls.pl.orig	2012-04-02 10:44:49.467776443 +0300
+++ lib/ssqls.pl	2012-04-02 10:45:19.753875195 +0300
@@ -32,7 +32,7 @@
 # will make ssqls.h exponentially larger.  This will increase compile
 # times and may even expose limits in your compiler.  Increase it only
 # if and as far as you must.
-my $max_data_members = 25;
+my $max_data_members = 71;
 
 # To make comparisons between floating point values, we subtract them,
 # take the absolute value, and test to see if that delta is under this
